packages/edit-site/src/components/navigation-sidebar/navigation-toggle/test/index.js

```diff
@@ -40,12 +90,23 @@ describe( 'NavigationToggle', () => {
 
 			render( <NavigationToggle /> );
 
+			// The `NavigationToggle` component auto-focuses on mount, and that
+			// causes the button tooltip to appear and to be positioned relative
+			// to the button. It happens async and we need to wait for it.
+			await waitFor(
+				() =>
+					expect(
+						screen.getByText( 'Toggle navigation' ).parentElement
+					).toBePositioned(),
+				{ timeout: 2000 } // It might take more than a second to position the popover.
```

Comment:
One little idea: maybe an `{ interval: 10 }` option is better. It better expresses the intent: we don't really need to wait longer, we only need to check more often 🙂

Reply:
Go for it 👍 Maybe the comment needs some improvements as well, to reflect that it's not about time but it's rather about the `waitFor()` loops ran under the hood.

Reply:
Implemented, including a comment update. :+1: